gooddata-flight-server/gooddata_flight_server/flexfun/flex_fun_execution_context.py

```diff
@@ -0,0 +1,105 @@
+# (C) 2024 GoodData Corporation
+from dataclasses import dataclass
+from typing import Optional
+
+from gooddata_sdk import Attribute, ComputeToSdkConverter, Filter, Metric
+
+
+@dataclass
```

Comment:
Consider using attrs instead.

Reply:
Let's not add this dependency until we need some of its features